case-lib/lib.sh

```diff
@@ -155,3 +155,22 @@ func_lib_lsof_error_dump()
         echo "$ret"
     fi
 }
+
+func_lib_get_tplg_path()
+{
+    local tplg=$1
+    # tplg given is empty
+    if [[ -z "$tplg" ]]; then
+        return 1
+    fi
+
+    if [[ -f "$tplg" ]]; then
+        echo "$tplg"
+    elif [[ -f "$TPLG_ROOT/$(basename "$tplg")" ]]; then
+        echo "$TPLG_ROOT/$(basename "$tplg")"
+    else
+        return 1
+    fi
```

Comment:
here will cause a hide bug, so please modify "$TPLG_ROOT/$(basename "$tplg")" as 1st order

Reply:
you can catch it with commit: https://github.com/thesofproject/sof-test/commit/dfa4d610a8161218ad6bbf64f947594ae938fec9#diff-c6a24841ec0a3d0dccc065831f99f31a

Reply:
thanks, fixed.